src/Compilers/CSharp/Portable/Binder/Semantics/BestTypeInferrer.cs

```diff
@@ -68,12 +68,12 @@ public static TypeSymbol InferBestType(ImmutableArray<BoundExpression> exprs, Co
         public static TypeSymbol InferBestTypeForConditionalOperator(BoundExpression expr1, BoundExpression expr2, Conversions conversions, out bool hadMultipleCandidates, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
         {
             // SPEC:    The second and third operands, x and y, of the ?: operator control the type of the conditional expression. 
-            // SPEC:    •	If x has type X and y has type Y then
-            // SPEC:        o	If an implicit conversion (§6.1) exists from X to Y, but not from Y to X, then Y is the type of the conditional expression.
-            // SPEC:        o	If an implicit conversion (§6.1) exists from Y to X, but not from X to Y, then X is the type of the conditional expression.
-            // SPEC:        o	Otherwise, no expression type can be determined, and a compile-time error occurs.
-            // SPEC:    •	If only one of x and y has a type, and both x and y, are implicitly convertible to that type, then that is the type of the conditional expression.
-            // SPEC:    •	Otherwise, no expression type can be determined, and a compile-time error occurs.
+            // SPEC:    •    If x has type X and y has type Y then
```

Comment:
These used to be aligned. I think it looks better this way, but just pointing out potentially unexpected consequences.